README.md

```diff
@@ -165,7 +165,8 @@ Capability | Description
 |----------|-----------|------|
 |`appium:orientation`|Start a test in a certain orientation|`LANDSCAPE` or `PORTRAIT`|
 |`appium:scaleFactor`|Simulator scale factor. This is useful to have if the default resolution of simulated device is greater than the actual display resolution. So you can scale the simulator to see the whole device screen without scrolling.|Acceptable values for simulators running Xcode SDK 8 and older are: `'1.0', '0.75', '0.5', '0.33' and '0.25'`, where '1.0' means 100% scale. For simulators running Xcode SDK 9 and above the value could be any valid positive float number. The capability must be of a string type.|
-|`appium:connectHardwareKeyboard`|Set this option to `true` in order to enable hardware keyboard in Simulator. It is set to `false` by default, because this helps to workaround some XCTest bugs.|`true` or `false`|
+|`appium:connectHardwareKeyboard`|Set this option to `true` in order to enable hardware keyboard in Simulator. The preference works only when Appium launches a simulator instance with this value. It is set to `false` by default, because this helps to workaround some XCTest bugs.|`true` or `false`|
+|`appium:forceTurnOnSoftwareKeyboard`|Set this option to `true` in order to turn software keyboard on since Appium 1.22.0. This option helps to avoid `Keyboard is not present` error. It is set to `true` by default. Appium respect preset simulator software/hardware keyboard preference when this value is `false`. This preference is prior rather than `connectHardwareKeyboard`. |`true` or `false`|
```

Comment:
I'm still not fully convinced on why we need two capabilities here, while we could only keep `connectHardwareKeyboard `